02-cpu-configure/Makefile

```
@@ -0,0 +1,9 @@
+CFLAGS = -Wall -Werror -O2
+
+all: pcie-remove
+
+pcie-remove: pcie-remove.c
+	cc $(CFLAGS) $< -o $@
+
+clean:
+	rm -f *.o pcie-remove
```

This should also work, using the default rules.

```suggestion

clean:
	rm -f pcie-remove
```